ark/utils/spatial_analysis_utils.py

```diff
@@ -12,64 +13,69 @@
 from tqdm.notebook import tqdm
 
 import ark.settings as settings
-from ark.utils import io_utils, misc_utils
+from ark.utils import io_utils, load_utils, misc_utils
 from ark.utils._bootstrapping import compute_close_num_rand
 
 
-def calc_dist_matrix(label_maps, save_path=None):
-    """Generate matrix of distances between center of pairs of cells
+def calc_dist_matrix(label_dir, save_path, prefix='_feature_0'):
```

Comment:
Is there logic to make sure this isn't run if the file exists already elsewhere, or should that be included here?

Reply:
We check this in both `example_pairwise_spatial_enrichment.ipynb` and `example_neighborhood_analysis_script.ipynb`. We create `dist_mat_dir` (synonymous with `save_path` here) and populate it only if it doesn't already exist.